### ISSUE TITLE
Doesn't always work with OpticManager which changes day length

### ISSUE THREAD

**Issue by BaccarWozat** [Quetzi/Morpheus#66]
My server has been getting no Morpheus action. With two people on the server at once, one sleeps and just stays in bed with nothing happening until the other also sleeps. Then it's regular vanilla sudden daytime with no sleep message.

This could possibly be caused by not recognizing the day length changed by OpticManager, which I have set to 30 minutes day / 30 minutes night.

Forge 2838, CreativeCore 1.9.45, IGCM 1.2.10, OpticManager 1.1.6, Morpheus 3.5.106.

**Comment by CreativeMD** [comment]
To explain a little bit how OpticManager works. It does not really increase the day or night but instead uses the total world tick time to manipulate the current tick time of the world to match the current state of the day. So if the configure day length is higher than the vanilla day, certain tick times occur twice (or more), if it's lower certain tick times will be skipped ([source](https://github.com/CreativeMD/OpticManager/blob/1.11/src/main/java/com/creativemd/opticmanager/OpticEventHandler.java#L94)).

**Comment by CreativeMD** [comment]
No, I do not modify the world settings (as explained above). The time of the world will keep cycling from 0 to 24000.

**Comment by Quetzi** [comment]
Then it should work fine, I suspect the issue is down to the configuration of the sleep vote. This specific version of Morpheus has had over 6 million downloads and the logic is unchanged from the previous version which had over 10 million downloads so I'm confident that there is no logic error on my side as this is the first report of this kind from 16.5 million installs.

**Comment by CreativeMD** [comment]
Yeah, if vanilla beds work yours should work as well. What could cause Morpheus to not display the text messages (maybe the issue is not related to the time)?

**Comment by Quetzi** [comment]
It's likely related to the config setting, but if you are cancelling tick events that could break all kinds of things, not just in Morpheus.

**Comment by CreativeMD** [comment]
I'm not cancelling any events, but world time might "jump" or "stay" between ticks. For example if the length of the day is configured to 48000, each tick time will stay for two ticks (it just overwrites the time).
